src/main/scala/edu/berkeley/ce/sparkrocks/LinearProgram.scala

```diff
@@ -21,9 +26,10 @@ object LinearProgram {
   * @constructor Initialize a new linear program.
   * @param numVars The number of variables in the linear program.
   */
-class LinearProgram(val numVars: Int) {
-  val solver = LpSolve.makeLp(0, numVars)
-  solver.setVerbose(LpSolve.IMPORTANT)
+class LinearProgram(numVars: Int) {
```

Comment:
We don't strictly need `val` in front of `numVars`, so I have removed it. I actually had to look up some of the precise rules for constructor parameters. See here for more info if you're interested: http://stackoverflow.com/questions/14694712/do-scala-constructor-parameters-default-to-private-val